pos_journal_pin/static/description/index.html

```diff
@@ -0,0 +1,107 @@
+      </p>		
+    </div>
+    <div class="oe_screenshot" align="center">
+      <img style="max-width: 70%" src="payment.png"/>
+    </div>
+  </div>
+</section>  
+    
+<section class="oe_container">
+    <div class="oe_row oe_spaced">
+        <div class="oe_span8">
+            <h2>Need our service?</h2>
+            <p class="oe_mt32">Contact us by <a href="mailto:apps@it-projects.info">email</a> or fill out <a href="https://www.it-projects.info/page/website.contactus " target="_blank">request form</a></p>
+            <ul>
+                <li><a href="mailto:apps@it-projects.info">apps@it-projects.info <i class="fa fa-envelope-o"></i></a></li>
+                <li><a href="https://www.it-projects.info/page/website.contactus " target="_blank">https://www.it-projects.info/page/website.contactus <i class="fa fa-list-alt"></i></a></li>
+                <li><a href="https://m.me/itprojectsllc" target="_blank">https://m.me/itprojectsllc <i class="fa fa-facebook-square"></i></a></li>
+                <li>skype@it-projects.info <i class="fa fa-skype"></i></li>
+            </ul>
+        </div>
+        <div class="oe_span4">
+            <div class="stamp" style="width:200px;">
+                <div style="margin-top: 15px;
+                            position: relative;
+                            font-family:'Vollkorn', serif;
+                            font-size: 16px;
+                            line-height: 25px;
+                            text-transform: uppercase;
+                            font-weight: bold;
+                            color: #75526b;
+                            border: 3px dashed #75526b;
+                            float: left;
+                            padding: 4px 12px;
+                            -webkit-transform: rotate(8deg);
+                            -o-transform: rotate(8deg);
+                            -moz-transform: rotate(8deg);
+                            -ms-transform: rotate(8deg);">
+                    Tested on Odoo<br/>10.0 community
+                </div>
+                <!--<div style="margin-top: 15px;
+                            position: relative;
+                            font-family:'Vollkorn', serif;
+                            font-size: 16px;
+                            line-height: 25px;
+                            text-transform: uppercase;
+                            font-weight: bold;
+                            color: #75526b;
+                            border: 3px dashed #75526b;
+                            float: left;
+                            padding: 4px 12px;
+                            -webkit-transform: rotate(6deg);
+                            -o-transform: rotate(6deg);
+                            -moz-transform: rotate(6deg);
+                            -ms-transform: rotate(6deg);">
+                    Tested on Odoo<br/>10.0 enterprise
+                </div>-->
+            </div>
+        </div>
+    </div>
+</section>
```

Comment:
No newline at and of File, look at the red sign after the tag

Reply:
done